### DISCUSSION TITLE
error: bundle argument "/tmp/buildah-TEST/" differs from inferred bundle "/tmp/buildah-TEST"

### DISCUSSION THREAD

**Discussion by naterini** [hpc/charliecloud#1114] [General]
I get this rather perplexing error while trying to test using  ch-run-oci create:
```
$ /usr/local/bin/ch-run-oci create --bundle /tmp/buildah-TEST/ --pid-file /tmp/buildah-test/pid buildah-buildah-TEST
error: bundle argument "/tmp/buildah-TEST/" differs from inferred bundle "/tmp/buildah-TEST"
```

ch Version:
> commit f1f99aa16eae73da0d3ce083de0cfa5c68203da5 (HEAD, tag: v0.24)

**Comment by reidpr** [comment]
It's just doing a simple string comparison; the argument to `--bundle` is `/tmp/buildah-TEST/` (trailing slash) while the inferred bundle path is `/tmp/buildah-TEST` (no trailing slash).

**Comment by reidpr** [comment]
Note `ch-run-oci` is a kludge and has only ever been tested with Buildah. If you're trying to use it in a different way it will almost certainly not work in its current state.

What are you trying to do?

I'm also happy to set up a meeting if that helps. E-mail me to set this up: reidpr@lanl.gov

**Comment by naterini** [comment]
> It's just doing a simple string comparison; the argument to `--bundle` is `/tmp/buildah-TEST/` (trailing slash) while the inferred bundle path is `/tmp/buildah-TEST` (no trailing slash).

Removing the slash got it to work.
